RecoMuon/MuonIdentification/plugins/MuonIdProducer.cc

```diff
@@ -637,6 +636,10 @@ void MuonIdProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
      }
    }
 
+   // muon arbitration
+   fillArbitrationInfo( outputMuons.get() );
+   arbitrateMuons( outputMuons.get(), caloMuons.get() );
```

Comment:
please use fillDescriptions to set a default value for a parameter.
The default can be overridden by specifying the parameter in a module instance where it's needed.